packages/react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts

```diff
@@ -63,7 +63,6 @@ export const getMenuItemStyles = memoizeFunction(
             // ensure disabled text looks different than enabled
             color: 'GrayText',
             opacity: 1,
-            ...getHighContrastNoAdjustStyle(),
```

Comment:
since we're setting GrayText, we don't need to override high contrast colors on the root when disabled.